src/components/pages/ProblemDetailedPage/styles.tsx

```diff
@@ -0,0 +1,181 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+import { ReactComponent as ArrowLeft } from "../../../assets/img/icons/arrow-left.svg";
+import { blue100, grey300 } from "../../../assets/styles/colors";
+import { minDevice } from "../../../assets/styles/breakpoints";
+import Wrapper from "../../atoms/Wrapper";
+
+const ProblemDetailedWrapper = styled(Wrapper)`
+  background: white;
+  flex-direction: column;
+  margin: 1.6rem 0 8rem 0;
+  min-height: 100vh;
+
+  @media ${minDevice.desktopTablet} {
+    margin-bottom: 2.1rem;
+  }
+
+  @media ${minDevice.desktopL} {
+    margin: 3.2rem 0 6.4rem 0;
+  }
+`;
+
+const GoToBackWrapper = styled(Link)`
```

Comment:
GoBackWrapper or GoToProblemsListWrapper?

Reply:
Yes, GoToProblemsListWrapper would be better, I'll rename it :)

Reply:
and also GoToProblemsListSpan

Reply:
Also the other place could be renamed :)